fundingmanager.go

```diff
@@ -1165,6 +1165,8 @@ func (f *fundingManager) handleFundingCreated(fmsg *fundingCreatedMsg) {
 				return
 			}
 
+			f.deleteReservationCtx(peerKey, fmsg.msg.PendingChannelID)
```

Comment:
I was running into a race otherwise where the channel would be marked as active in the db, but [this check](https://github.com/lightningnetwork/lnd/blob/master/fundingmanager.go#L711) was reporting that there were still pending channels.